src/coreclr/src/jit/stacklevelsetter.cpp

```diff
@@ -55,6 +59,9 @@ void StackLevelSetter::DoPhase()
 
     comp->fgSetPtrArgCntMax(maxStackLevel);
     CheckArgCnt();
+
+    // Might want an "other" category for things like this...
```

Comment:
Any idea what checking such an "other" category might want to trigger?

Reply:
Not sure? Seems like a good part of the compiler object state probably can't be checked effectively. But maybe there are some things we can come up with.